dominicstop/react-native-ios-context-menu
# chore: upgrade example-expo to SDK 52

This PR upgrades the Expo example to SDK 52 by [following this walkthrough](https://docs.expo.dev/workflow/upgrading-expo-sdk-walkthrough/), and then the [upgrade helper](https://docs.expo.dev/bare/upgrade/?fromSdk=51&toSdk=52#androidappsrcmainjavacomhelloworldmainapplicationkt) since we aren't using [continuous native generation](https://docs.expo.dev/workflow/continuous-native-generation/). I also updated the `react-native-ios-utilities` dependency to `5.0.0-53`

I also turned on the new architecture by default in the Expo example.

To test this out:

1. Clean your build folders with `yarn nuke:all` (or some less intense process if you don't want to nuke your whole machine).
2. Run `yarn install`
3. `cd example-expo/ios && pod install` (you may need to run with `--repo-update`, but hopefully the `Podfile.lock` changes make that unnecessary)
4. Run `cd .. && yarn ios` to run the iOS Expo build.
5. Run `yarn android` to run the Android Expo build

The app should build and run as expected (although the Android app of course doesn't do anything, other than build and show text).
